tuning/config/tracker_configs.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+
+@dataclass
+class AimConfig:
+    # Name of the experiment
+    experiment: str = None
+    # 'repo' can point to a locally accessible directory (e.g., '~/.aim') or a remote repository hosted on a server.
```

Comment:
might be clearer to use the actual config name in the description (`aim_repo`)... though the `aim_` in the config names might be a bit redundant too.

Reply:
well taken..this was a typo..fixed.